### ISSUE TITLE
Add Set Transformer to the models

### ISSUE THREAD

**Issue by Sarasra** [tensorflow/graphics#318]
Add a Tensorflow implementation of the paper [Set Transformer: A Framework for Attention-based Permutation-Invariant Neural Networks](http://proceedings.mlr.press/v97/lee19d.html). 
They achieve .84, .89 and .90 percent accuracy on Modelnet40.

**Comment by osanseviero** [comment]
Hi @Sarasra! Is this something you would like to implement?
